cmd/lotus-storage-miner/retrieval-deals.go

```diff
@@ -154,3 +159,112 @@ var retrievalDealsListCmd = &cli.Command{
 		return w.Flush()
 	},
 }
+
+var retrievalSetAskCmd = &cli.Command{
+	Name:  "set-ask",
+	Usage: "Configure the provider's retrieval ask",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "price",
+			Usage: "Set the price of the ask for retrievals (per byte)",
+		},
+		&cli.StringFlag{
+			Name:  "unseal-price",
+			Usage: "Set the price to unseal",
+		},
+		&cli.StringFlag{
+			Name:        "payment-interval",
+			Usage:       "Set the payment interval (in bytes) for retrieval",
+			DefaultText: "1Mb",
```

Comment:
Nit: units.RAMInBytes will parse this as 2^20, so technically we should be saying `1MiB`